migrations/01-create-artist-table.sql

```diff
@@ -0,0 +1,5 @@
+ CREATE TABLE Artists (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255) NOT NULL,
+    genre VARCHAR(255) NOT NULL
+    );
```

Comment:
Just indentation here. Remember you should have all the contents of a function indented by one further layer than the function opening, and the closing bracket should be at the same level as the opening one
```
  id SERIAL PRIMARY KEY,
  name VARCHAR(255) NOT NULL,
  genre VARCHAR(255) NOT NULL
);
```